installer-static.tpl.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# This script is used to install the required packages for the application
```

Comment:
I think this comment is a little strange. What required packages for what application?
Consider this one: 
```
Script to set up the repository for Tarantool version >= 3.
Provide TARANTOOL_AUTO_INSTALL=True to the script to automatically
install Tarantool after the repository setup.
```

Reply:
OK